penaltymodel_mip/setup.py

```diff
@@ -8,7 +8,7 @@
 
 install_requires = ['dimod>=0.6.0,<0.10.0',
                     'networkx>=2.0,<3.0',
-                    'ortools>=6.6.4659,<8.0.0',
```

Comment:
it makes me a bit nervous to remove the upper bound entirely. They change their API semi often.

Reply:
👍  I can make it <8.1.0 ?

Reply:
If an 8.x is working, then I think `<9.0.0`

Reply:
Done